src/main/java/uk/gov/hmcts/ccd/data/draft/DefaultDraftGateway.java

```diff
@@ -77,7 +77,7 @@ public Long save(final CreateCaseDraft draft) {
             return getDraftId(responseHeaders);
         } catch (Exception e) {
             LOG.warn("Error while saving draft=" + draft, e);
-            throw new ServiceException("Problem saving draft because of " + e.getMessage());
+            throw new ServiceException(DRAFT_STORE_DOWN_ERR_MESSAGE);
```

Comment:
Even if we are logging the Exception e as a WARN, 
I think we should add the whole Exception e as a a caused by when we create the ServiceException. This way when later we log the error with ERROR level we get the 'caused by' and we know what the real issue is. 
ServiceException at the moment doesn't support passing a caused by, but we can add that.

Reply:
we could do this in a tech debt story, as I see we are doing this everywhere